common/persistence/cassandra/cassandraQueue.go

```diff
@@ -206,15 +224,59 @@ func (q *cassandraQueue) ReadMessages(
 	return result, nil
 }
 
-func getMessagePayload(message map[string]interface{}) []byte {
+func (q *cassandraQueue) ReadMessagesFromDLQ(
+	lastMessageID int,
+) ([]*persistence.QueueMessage, error) {
+	// Reading replication tasks need to be quorum level consistent, otherwise we could loose task
+	// Use negative queue type as the dlq type
+	query := q.session.Query(templateGetMessagesFromDLQQuery,
+		-q.queueType,
+		lastMessageID,
+	)
+
+	iter := query.Iter()
```

Comment:
what is the page size here ? Are you simply going to return all the messages from DLQ in one shot ?

Reply:
Add the max count